### ISSUE TITLE
Partial rotation should be described before reserve rotation.

### ISSUE THREAD

**Issue by nkongsuwan** [trustoverip/tswg-keri-specification#130]
Currently, the section on [partial rotation](https://trustoverip.github.io/tswg-keri-specification/#partial-pre-rotation) comes after the section on reserve rotation [reserve rotation](https://trustoverip.github.io/tswg-keri-specification/#reserve-rotation). I suggest that Partial Rotation should be the main section while reserve rotation should be its sub-section.

**Comment by m00sey** [comment]
Partial rotation is no longer a super set of reserve rotation - the definition has changed - let me know if this works for you @nkongsuwan

**Comment by nkongsuwan** [comment]
@m00sey Thank you for the response. In that case, I think we need to change [Section 8.6](https://trustoverip.github.io/tswg-keri-specification/#reserve-rotation) slightly. 

The first sentence of this section is `The pre-rotation mechanism supports partial pre-rotation or, more exactly, partial Rotation of pre-rotated keypairs`, but `partial rotation` is later introduced in [Section 8.6.2](https://trustoverip.github.io/tswg-keri-specification/#partial-pre-rotation).

If partial rotation is no longer a superset of reserve rotation, the section on `reserve rotation` should be described more generally. Then, the `partial rotation` can be described as the special case of `reserve rotation` or vice versa.

Note: I might be able to work on a PR for this issue next week if no one gets to this before me.

**Comment by nkongsuwan** [comment]
Reading the specification as it is (18 Feb), it implies that partial rotation is still a superset of reserve rotation. In my opinion, it makes more sense to swap the order of these sections to 
```
8.6 Partial pre-rotation
8.6.1 Reserve rotation
8.6.2 Custodial rotation
```

@m00sey would you be able to clarify what is the new definition for reserve rotation?

**Comment by SmithSamuelM** [comment]
I fixed this with #155.
Thanks @nkongsuwan   it looks like to me that somewhere in organization of the this version of the spec that the paragraphs got misordered because clearly the partial rotation should be first and reserve and custodial are subsets.
That is now fixed.
